cfme/fixtures/cli.py

```diff
@@ -407,29 +454,80 @@ def ha_appliances_with_providers(ha_multiple_preupdate_appliances, app_creds):
     interaction.answer('Enter the cluster database username: |root| ', '')
     interaction.answer('Enter the cluster database password: ', pwd)
     interaction.answer('Enter the cluster database password: ', pwd)
-    interaction.answer('Enter the primary database hostname or IP address: ', app0_ip)
-    interaction.answer('Enter the Standby Server hostname or IP address: |.*|', app1_ip)
+    interaction.answer('Enter the primary database hostname or IP address: ', primary_ip)
+    interaction.answer('Enter the Standby Server hostname or IP address: |.*|', appl.hostname)
     interaction.answer(r'Configure Replication Manager \(repmgrd\) for automatic '
                        r'failover\? \(Y/N\): ', 'y')
     interaction.answer(r'Apply this Replication Server Configuration\? \(Y/N\): ', 'y')
     interaction.answer('Press any key to continue.', '', timeout=5 * 60)
 
+
+def reconfigure_standby_replication_node(appl, pwd, primary_ip, repmgr_reconfigure=False):
```

Comment:
Thes `reconfigure*` method are similar to the `configure*` ones. Perhaps it would be worthy to somehow dedup them